quantum/matrix.c

```diff
@@ -194,114 +194,61 @@ uint8_t matrix_key_count(void)
 
 static void init_cols(void)
 {
-    int B = 0, C = 0, D = 0, E = 0, F = 0;
-
 #if DIODE_DIRECTION == COL2ROW
-    for(int x = 0; x < MATRIX_COLS; x++) {
-        int col = COLS[x];
+    for (uint16_t col = 0; col < MATRIX_COLS; col++) {
+        const uint8_t col_pin = COL_PINS[col];
 #else
-    for(int x = 0; x < MATRIX_ROWS; x++) {
-        int col = ROWS[x];
+    for (uint16_t col = 0; col < MATRIX_ROWS; col++) {
+        const uint8_t col_pin = ROW_PINS[col];
```

Comment:
Done. I think we should keep `COL2ROW`and `ROW2COL` as they are—people not familiar not familiar with how the matrix works might have trouble figuring out which of those to define.